packages/sacado/test/UnitTests/Fad_Kokkos_CommTests_Serial.cpp

```diff
@@ -51,19 +51,19 @@ typedef Sacado::ELRCacheFad::SLFad<double,10> ELRCacheFad_SLFadType;
 typedef Sacado::ELRCacheFad::SFad<double,5> ELRCacheFad_SFadType;
 Sacado::Random<double> rnd;
 
-FAD_KOKKOS_COMM_TESTS_SERIAL(Fad_DFadType, Fad_DFad)
+//FAD_KOKKOS_COMM_TESTS_SERIAL(Fad_DFadType, Fad_DFad)
 FAD_KOKKOS_COMM_TESTS_SERIAL(Fad_SLFadType, Fad_SLFad)
 FAD_KOKKOS_COMM_TESTS_SERIAL(Fad_SFadType, Fad_SFad)
 
-FAD_KOKKOS_COMM_TESTS_SERIAL(CacheFad_DFadType, CacheFad_DFad)
+//FAD_KOKKOS_COMM_TESTS_SERIAL(CacheFad_DFadType, CacheFad_DFad)
 FAD_KOKKOS_COMM_TESTS_SERIAL(CacheFad_SLFadType, CacheFad_SLFad)
 FAD_KOKKOS_COMM_TESTS_SERIAL(CacheFad_SFadType, CacheFad_SFad)
 
-FAD_KOKKOS_COMM_TESTS_SERIAL(ELRFad_DFadType, ELRFad_DFad)
+//FAD_KOKKOS_COMM_TESTS_SERIAL(ELRFad_DFadType, ELRFad_DFad)
 FAD_KOKKOS_COMM_TESTS_SERIAL(ELRFad_SLFadType, ELRFad_SLFad)
 FAD_KOKKOS_COMM_TESTS_SERIAL(ELRFad_SFadType, ELRFad_SFad)
 
-FAD_KOKKOS_COMM_TESTS_SERIAL(ELRCacheFad_DFadType, ELRCacheFad_DFad)
+//FAD_KOKKOS_COMM_TESTS_SERIAL(ELRCacheFad_DFadType, ELRCacheFad_DFad)
 FAD_KOKKOS_COMM_TESTS_SERIAL(ELRCacheFad_SLFadType, ELRCacheFad_SLFad)
 FAD_KOKKOS_COMM_TESTS_SERIAL(ELRCacheFad_SFadType, ELRCacheFad_SFad)
 
```

Comment:
Why are the DFad tests disabled?

Reply:
Oh Shoot! Good catch. Thanks! This was from working around the "initialization bug" that turned out to be the corrupted node on caraway. I will reenable these asap.

Reply:
tests are reenabled